awx/main/tests/unit/test_unified_jobs.py

```diff
@@ -8,15 +8,15 @@
 from django.utils.timezone import now
 
 # AWX
-from awx.main.models import UnifiedJob
+from awx.main import models
 
 
 # stdout file present
 @mock.patch('os.path.exists', return_value=True)
 @mock.patch('codecs.open', return_value='my_file_handler')
+@mock.patch.object(models.UnifiedJob, 'result_stdout_text', '')
```

Comment:
These tests have changed to just mock the `result_stdout_text` attribute access; they don't need to care about the underlying implementation details of how the data gets stored and retrieved.